Aliases/ignition-tools1

```diff
@@ -1 +1 @@
-../Formula/ignition-tools.rb
```

Comment:
this change should be reverted

Reply:
https://github.com/osrf/homebrew-simulation/pull/1888/commits/cdfd567ea7eac97ff4dd046005076b185e05408b